Libraries/Cordio/docs/CORDIO_USERGUIDE.md

```diff
@@ -0,0 +1,276 @@
+
+Devices in the scanning state listen for advertising devcies and can send scan optionally requests for additional information. The scanning interval and window settings are configurable. 
+
+If a devices is scanning with the intent of connecting to a specific device, that is called the initiating state. The initiating devcies will send a connection indication to the desired advertising device to indicate it's desire to create a connection.
+
+#### Connected
+
+Once a initiating devices sends the connection requset and the advertising devices accepts the connection request, the two devices enter the connected state. This is a point to point connection allowing devcies to directly exchange information. 
+
+In order to minimize interference, devices in the connected state will hop between channels 0-36 in a pseudo random order. The channel hopping information is communicated in the connection indication.
+
+Each connection event is separated by the connection interval. This interval is configurable from 7.5 ms to 4 s. The master will always transmit first and receive second. The Slave will always receive first and transmit second. Devices will typically always send and receive at least one packet in each interval, and they can optionally transmit and receive multiple packets in each interval. 
+
+![Connected](res/Connected.png)
+
+## Packetcraft Protocol Software
+
+Packetcraft protocol software is a collection of embedded stacks implementing the Bluetooth Low Energy Link Layer, Host, Profile and Mesh specification (www.bluetooth.org).
+
+This repository contains open source release of Packetcraft's software. This is a qualified release and may be used in products. Please consult the [Bluetooth Qualification Process](https://www.bluetooth.com/develop-with-bluetooth/qualification-listing) for further details regarding additional certification requirements.
+
+Release notes
+-------------
+
+This latest release of the Packetcraft Host and Packetcraft Controller is Bluetooth 5.2 qualified and implements the following new Bluetooth 5.2 features: LE Isochronous Channels, Enhanced Attribute Protocol, and LE Power Control.
+
+This release includes the following completed requirements for r20.05:
+
+    FW-3340 Isochronous Demo: single BIS data stream
+    FW-3354 TCRL.2019-1 compliant
+    FW-3359 Core v5.2: LE Isochronous Channels (ISO)
+    FW-3360 Core v5.2: LE Power Control
+    FW-3361 Core v5.2: Enhanced ATT (EATT)
+    FW-3617 Core v5.2: Isochronous Abstraction Layer (ISOAL)
+    FW-3726 Core v5.2: Host Support for LE Isochronous Channels (ISO)
+    FW-3727 Nordic nRF5 SDK 16.0.0
+    FW-3730 Compile BLE host for 64-bit platform
+    FW-3736 Light CTL Model
+    FW-3738 Mesh v1.0.1 compliant
+    FW-3739 TCRL.2019-2 qualification
+    FW-3750 Laird BL654 platform
+    FW-3767 SBC codec
+    FW-3803 GCC compiler support for gcc-arm-none-eabi-9-2019-q4-major
+    FW-3820 Protect against SweynTooth vulnerability
+
+
+Getting Started
```

Comment:
This section recommends manually installing GCC and has some broken links.  We should remove this entirely or rework it to point to the MSDK Getting Started sections and highlight some good BLE starter projects.